pkg/test/framework/components/istio/operator.go

```diff
@@ -102,6 +102,13 @@ var leaderElectionConfigMaps = []string{
 	leaderelection.ValidationController,
 }
 
+func (i *operatorComponent) Ingress(cluster resource.Cluster) ingress.Instance {
+	return ingress.New(i.ctx, ingress.Config{
```

Comment:
Do we want a new one every time? I would imagine usage is like `istio.Ingress().Call(..); istio.Ingress().Call(...)` and then we suddenly waste a lot of work (I think)

Reply:
Well I guess that isn't how you are using it - but others might. if you don't want to have a shared one returned (which is fine, I don't have a strong preference) can we put a comment on the interface?

Reply:
This was super WIP, should have been in draft mode.

Reply:
I changed this to do the memo but it actually looks like Call does all the real work for instantiating a client and doing a handshake.